src/booleans.js

```diff
@@ -1,62 +1,68 @@
 function negate(a) {
-  // your code here
+// NOTE GATE
+  return (!a);
 };
 
 function both(a, b) {
-  // your code here
+// AND GATE
+  return (a&&b);
 };
 
 function either(a, b) {
-  // your code here
+// OR GATE
+  return (a||b);
 };
 
 function none(a, b) {
-  // your code here
+// NOR GATE
+  return !(a||b);
 };
 
 function one(a, b) {
-  // your code here
+// XOR GATE
+  return (a && !b) || (!a && b);
 };
 
 function truthiness(a) {
-  // your code here
+  return Boolean(a);
 };
 
 function isEqual(a, b) {
-  // your code here
+  return a===b;
 };
 
 function isGreaterThan(a, b) {
-  // your code here
+return a>b;
 };
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
+return a<=b;
 };
 
 function isOdd(a) {
-  // your code here
+  // NUMBERS NOT DIVISIBLE BY 2, REMAINDER CAN'T BE 0!
+  return a%2 == !0;
 };
 
 function isEven(a) {
-  // your code here
+ // NUMBERS DIVISIBLE BY 2, REMAINDER MUST = 0!
+ return a%2 == 0;
```

Comment:
Comment should be deleted.
Add some spaces as above and use the strict quality (===) operator.
And the return statement is missing the indentation.